tools/paraclu/test/main.nf

```diff
@@ -40,14 +40,52 @@ Channel
     .map { row -> [ row[0], file(row[1], checkIfExists: true) ] }
     .set {ch_crosslinks}
 
+output_line_counts = [
+    Sample1: 7,
+    Sample2: 0,
+    Sample3: 0,
+    Sample4: 4,
+    Sample5: 0,
+    Sample6: 0
+]
+
 /*------------------------------------------------------------------------------------*/
 /* Run tests
 --------------------------------------------------------------------------------------*/
 
+process count_lines {
+    label "low_cores"
+    label "low_mem"
+    label "regular_queue"
+
+    tag "${sample_id}"
+
+    container 'ubuntu:16.04'
+
+    input:
+        tuple val(sample_id), path(peaks)
+
+    output:
+        tuple val(sample_id), stdout, emit: line_count
+
+    script:
+    """
+    echo -n "\$(gunzip -c $peaks | wc -l)"
+    """
+}
+
```

Comment:
We can change this to the test workflow once that PR goes through

Reply:
Yep, I've also written a version of the module which doesn't need a custom container, which I'll update to after we merge the other PR